packages/repack/src/webpack/federated.ts

```diff
@@ -44,10 +44,13 @@ export namespace Federated {
               return self.${remoteName}.get(request);
             },
             init: (arg) => {
-              try {
-                return self.${remoteName}.init(arg);
-              } catch (e) {
-                console.log('[Repack/Federated] Remote container ${remoteName} already initialized.');
+              if(!self.${remoteName}._initialized) {
+                try {
+                  self.${remoteName}._initialized = true
+                  return self.${remoteName}.init(arg);
+                } catch (e) {
+                  console.log('[Repack/Federated] Remote container ${remoteName} already initialized.');
```

Comment:
good idea to still keep the try catch just incase someone loaded a remote in some non standard way & the initialized flag was not set